metrics/gauge.go

```diff
@@ -2,7 +2,7 @@ package metrics
 
 import "sync/atomic"
 
-// Gauges hold an int64 value that can be set arbitrarily.
+// Gauge methods hold an int64 value that can be set arbitrarily.
```

Comment:
Please don't add `methods` all over, it's a simple interface.

Reply:
Thank you - updated.